ci/recipe/meta.yaml

```diff
@@ -14,14 +14,14 @@ build:
 
 requirements:
   build:
```

Comment:
```suggestion
   host:
```
New conda uses build for compile toolchains, it's kind of backwards compatible right now, but may as well update it.